polygeist/tools/cgeist/Lib/clang-mlir.cc

```diff
@@ -1552,6 +1552,8 @@ ValueCategory MLIRScanner::CommonFieldLookup(clang::QualType CT,
 
     Result = Builder.create<polygeist::SubIndexOp>(Loc, ResultType, val,
                                                    getConstantIndex(fnum));
+  } else if (auto RT = MT.getElementType().dyn_cast<mlir::sycl::AtomicType>()) {
+    llvm_unreachable("not implemented");
```

Comment:
Can we implement this? Wouldn't copying the stuff above be enough?

Reply:
Done